dashboard/src/App.js

```diff
@@ -16,21 +16,30 @@ import { AuthForm } from "modules/components/AuthComponent/common-components";
 import AuthLayout from "modules/containers/AuthLayout";
 import ComingSoonPage from "modules/components/EmptyPageComponent/ComingSoon";
 import Cookies from "js-cookie";
-import LoginForm from "modules/components/AuthComponent/LoginForm";
 import MainLayout from "modules/containers/MainLayout";
 import NoMatchingPage from "modules/components/EmptyPageComponent/NoMatchingPage";
 import OverviewComponent from "modules/components/OverviewComponent";
 import ProfileComponent from "modules/components/ProfileComponent";
-import SignupForm from "modules/components/AuthComponent/SignupForm";
 import TableOfContent from "modules/components/TableOfContent";
 import TableWithFavorite from "modules/components/TableComponent";
 import favicon from "./assets/logo/favicon.ico";
 import { fetchEndpoints } from "./actions/endpointAction";
-import { getUserDetails } from "actions/authActions";
 import { showToast } from "actions/toastActions";
-import { useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
+import { ReactKeycloakProvider } from "@react-keycloak/web";
 
-const ProtectedRoute = ({ redirectPath = APP_ROUTES.AUTH_LOGIN, children }) => {
+const eventLogger = (event, error) => {
+  // Placeholder to perform logging action upon certain Keycloak events.
+  // Keycloak events are described under
+  // https://www.keycloak.org/docs/latest/securing_apps/#javascript-adapter-reference
+};
+
+const tokenLogger = (tokens) => {
+  // Placeholder for to perform action when new token is generated
+  // console.log('onKeycloakTokens', tokens["refreshToken"]);
+};
```

Comment:
Rather than include an empty logger hooks in the code, can we just leave them out? They can be rediscovered, and you could even add a comment near the `App` use to describe the logger hooks if it makes you feel better.